client/services/index.js

```diff
@@ -1,2 +1,3 @@
 export {PublisherFactory} from './PublisherFactory';
 export {PubAPIFactory} from './PubAPIFactory';
+export {PublisherHelpersFactory} from './PublisherHelpers';
```

Comment:
@tomaszrondio - it shouldn't be `'./publisherHelpers';`? As file is named `client/services/publisherHelpers.js`.